src/libDirectoryService/PoWProcessing.cpp

```diff
@@ -314,8 +314,17 @@ bool DirectoryService::ProcessPoWSubmissionFromPacket(
       return false;
     }
   } else {
-    if (difficultyLevel != expectedDSDiff && difficultyLevel != expectedDiff &&
-        difficultyLevel != expectedShardGuardDiff) {
+    bool difficultyCorrect = true;
+    if (Guard::GetInstance().IsNodeInShardGuardList(submitterPubKey)) {
```

Comment:
Can change to `if (GUARD_MODE && IsNodeInShardGuardList)` so that we don't get flooded with `Not in Guard mode` messages.

Reply:
On top already checked GUARD_MODE